### ISSUE TITLE
vcbuild stuck

### ISSUE THREAD

**Issue by xblox** [nexe/nexe#294]
Hi,
since some time, my windows build gets stuck forever (waited long!) at producing node.exe. I am building this inside a Virtual-Machine(windows-10) since long and never had problems. If I run vcbuild.bat manually: all ok, running node.sln with Visual-Studio: all ok too. 

I tried now Virtual-Machines with vs2015-2017 (with updates or not) but it always ends up the same way. So I guess: child_process.spawn or VMware has problems. 

Did you ever encountered this ? 

I am fighting this now since a week and I am really out of advice. 

close this just if you feel like.

**Comment by jaredallard** [comment]
I'm not a 100% sure what's going on here. Can you drop some Terminal output, logs, etc, or anything? Thanks.

**Comment by xblox** [comment]
![image](https://cloud.githubusercontent.com/assets/1205100/21962415/abb2bb8e-db25-11e6-9be0-4a65365ea24a.png)

hi, thanks for the heads up. Its always stuck there, for days. No matter its VMware, Virtual-Box or whatever new fresh win/v-studio installation. Also, I tried child_process.spawn with "detached" : same result. 

Gonna try a physical windows now.

**Comment by xblox** [comment]
Ok, had no windows but a osx physical setup (same problem with running nexe inside a virtual box/machine) around but I figured an work-around which also improving 

1. one - time only: spawn vcbuild with : nosign, release, **nobuild**
2. if config/project files ready: call msbuild instead of vcbuild

done

doing it like that also saves you from redundant builds if only the nexe.js was changed. 

However, I am doing this now in my fork, if you want it, let me know.
